mailbox/store/src/main/java/org/apache/james/mailbox/store/event/DefaultDelegatingMailboxListener.java

```diff
@@ -107,4 +107,13 @@ protected void deliverEventToGlobalListeners(Event event) {
         }
     }
 
+    @Override
+    public boolean isListeningTo(MailboxListener mailboxListener) {
```

Comment:
I don't understand this method : `this` is a listener and you ask if it listens another listener.
I mean, a listener usually listen to events, don't you have a typing problem here ?